docs/appendices/release-notes/unreleased.rst

```diff
@@ -113,15 +113,5 @@ Fixes
 .. stable branch. You can add a version label (`v/X.Y`) to the pull request for
 .. an automated mergify backport.
 
-- Fixed a file descriptor leak that was triggered by querying the ``os`` column
-  of the ``sys.nodes`` table.
+None
```

Comment:
Is it ok to remove them now? or when `5.0.2` is released?

Reply:
I treat it as branch specific - in master it's kinda general state (released in 4.8.4) and in 5.0 we still have those entries as they are not released in 5.0. @seut wdyt?

Reply:
I actually think now that I have to remove entries from 5.0, i.e make https://github.com/crate/crate/pull/13043 similar to this PR. as 5.0 should reflect current state of earlier branches

Reply:
Hm, on second thought, we shouldn't update 5.0, cause the upcoming 5.0.2 release should include the same fixes in the release notes, imho.

Reply:
>upcoming 5.0.2 release should include the same fixes 

makes sense, so keep in another PR (as it does) and remove from 4.8 here.

Reply:
This entries should not be removed at the `5.0` branch for sure as they indicate that this branch contains unreleased fixes. And I'd prefer to keep them here as well until we've released another `5.0` hotfix.

Reply:
reverted unreleased